From 3380986b972ad96a60775892493a65d8095c02ce Mon Sep 17 00:00:00 2001
From: Shilin Victor <chrono.monochrome@gmail.com>
Date: Sun, 26 Aug 2018 18:13:30 +0300
Subject: [PATCH 29/32] Revert ".KATI_obsolete PRODUCT_FULL_TREBLE."

This reverts commit 5dcea5de76ef0edb69b5f8aa71a99be707ccb324.

Change-Id: Ide379d9ecbaecf560ff7eadc98d7aaa7f4d036a3
---
 core/main.mk | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/core/main.mk b/core/main.mk
index 2e229764d..66740ca96 100644
--- a/core/main.mk
+++ b/core/main.mk
@@ -204,11 +204,6 @@ endif
 
 ADDITIONAL_BUILD_PROPERTIES += ro.treble.enabled=${PRODUCT_FULL_TREBLE}
 
-$(KATI_obsolete_var PRODUCT_FULL_TREBLE,\
-	Code should be written to work regardless of a device being Treble or \
-	variables like PRODUCT_SEPOLICY_SPLIT should be used until that is \
-	possible.)
-
 # Sets ro.actionable_compatible_property.enabled to know on runtime whether the whitelist
 # of actionable compatible properties is enabled or not.
 ifeq ($(PRODUCT_ACTIONABLE_COMPATIBLE_PROPERTY_DISABLE),true)
-- 
2.11.0

